README_zh-CN.md

```diff
@@ -316,7 +316,7 @@ MMAction2 是一款由不同学校和公司共同贡献的开源项目。我们
 扫描下方的二维码可关注 OpenMMLab 团队的 [知乎官方账号](https://www.zhihu.com/people/openmmlab)，加入 OpenMMLab 团队的 [官方交流 QQ 群](https://jq.qq.com/?_wv=1027&k=aCvMxdr3)
 
 <div align="center">
-<img src="https://github.com/open-mmlab/mmaction2/raw/master/resources/zhihu_qrcode.jpg" height="400" />  <img src="https://github.com/open-mmlab/mmaction2/raw/master/resources/qq_group_qrcode.jpg" height="400" />
+<img src="https://github.com/open-mmlab/mmaction2/raw/master/resources/zhihu_qrcode.jpg" height="400" />  <img src="https://github.com/duanzhihua/mmaction2/blob/master/resources/qq_group_qrcode.png" height="400" />
```

Comment:
Thanks for your contribution! We suggest using the link to the image in the open-mmlab/mmaction2 repo here.